### PR TITLE
TEMPORARILY removes the stock exchange machines from cargo

### PR DESCRIPTION
**Pull request by PrinceKirze**
steals cargos money; makes them broke

### Intent of your Pull Request

Removing the stock exchange machines from the maps that they are on because the current stock system is shit and makes balancing cargo a pain in the rear. Gamer is gonna rework it and re-add it in, I promise. 

### Why is this change good for the game?

Cargo having some money is good. Cargo having three hundred thousand credits and buying a shuttle's worth of riot shotguns is not. 

# Wiki Documentation

Could alter/remove mentions of the stock machine but it's going to get re-added so probably just a note that says it's currently removed so we don't get 50 mhelps about where the stock machine went

### Briefly describe your PR and the impacts of it, in layman's terms. 
stock machine go sleepy time for a second

### What should players be aware of when it comes to the changes your PR is implementing?
no stocks

### What general grouping does this PR fall under? 
Cargo balancing

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
no

# Changelog

:cl:  
rscdel: Temporarily removing stock machines from maps that have them
/:cl:

### CODE REVIEW

**Comment by PrinceKirze** [comment]
sloppy pr cuz i forgot how to do this but it looks fiine

**Comment by Tipy1802** [comment]
Just how budget cards were temporaly removed :^)

**Comment by PrinceKirze** [comment]
> Just how budget cards were temporaly removed :^)

And then the necessary one re-added

**Review by patpol4 — CHANGES_REQUESTED** [review]
Pretty sure there are two of these in box cargo, also dont forget meta, delta, and omega.
This is a balance thing so I'll leave it up to HC after you do it properly

**Comment by PrinceKirze** [comment]
> Pretty sure there are two of these in box cargo, also dont forget meta, delta, and omega.
> This is a balance thing so I'll leave it up to HC after you do it properly

couldn't find any on those three maps, but i think i did miss the second one in box.

**Review by alexkar598 — COMMENTED** [review]
You realize they are craftable?

**Comment by Hopekz** [comment]
There is no reason to remove them round start now vs when a rework happens.
PR's based on someone else's future promises are lame

**Comment by PrinceKirze** [comment]
I'll fix all the shit tomorrow. Forgot they were craftable and shit came up today. 

> There is no reason to remove them round start now vs when a rework happens.
> PR's based on someone else's future promises are lame

There's as much a reason to remove it as there is to remove anything else that is fucked beyond belief. The promise for a Gamer rework is more a reassurance that people who like the idea of stocks will get something else in a bit, but Gamer likes to try and meme this shit into bounties so I don't doubt this will take a little while; hence, the removal for now.

**Comment by ghost** [comment]
@Tipy1802 Good thing budget cards are being permanently removed

**Comment by maxion12345** [comment]
gamer is reworking stocks?

but...can we fix zombies first tho

**Comment by SomeguyManperson** [comment]
this isn't based on future changes stocks are busted and either need to be reworked or removed
the only time stocks are actually useful due to the way their code functions is when they break entirely and skyrocket to the maximum price

**Review by patpol4 — CHANGES_REQUESTED** [review]
Apparently these exist just on box and eclipse, so just remove the other one from box and this will be good from a map side of things

**Review by patpol4 — APPROVED** [review]
Now it's fine mapwise

**Comment by JamieD1** [comment]
Now the design is gone too, it actually does something.

**Comment by JamieD1** [comment]
If something is not in place within a month this is being reverted.

07/02/2020

**Comment by PrinceKirze** [comment]
> If something is not in place within a month this is being reverted.
> 
> 07/02/2020

imagine using this date format 0_o
